FinSearchUnified/Components/ConfigLoader.php

```diff
@@ -71,7 +70,7 @@ private function isSuccessful(Response $response)
     {
         $restype = (string)$response->getStatusCode();
 
-        return $restype[0] === '2' || $restype[0] === '1';
+        return strpos($restype, '2') === 0 || strpos($restype, '1') === 0;
```

Comment:
If we are already refactoring this code, we could also cast the status code to an integer and ensure that it is bigger or equal 100 and lower than 300.
Please add unit-tests for code that you have changed to ensure your change does still work as expected.

Reply:
The logic is still the same for this change, its just a bit of optimization in terms of readability

Reply:
I just checked the `ConfigLoaderTest` and it seems like this method is already covered in tests. For future changes please keep in mind that even if it is a small change or even just a change that makes the code more readable, there should always be an unit-test associated to it. Because if something goes wrong (for whatever reason), the build will break and will notify us.

Reply:
Sure will keep that in mind for future. Although I knew that the case was already covered as I wrote that myself. I was about to tell you about it but got occupied in some other task